### PR TITLE
DPC-1175: Static Site Set Up

### PR DESCRIPTION
**Pull request by Sun-Mountain**
### Fixes [DPC-1174](https://jiraent.cms.gov/browse/DPC-1174) & [DPC-1175](https://jiraent.cms.gov/browse/DPC-1175)

The static pages and docker container are migrating from my personal repo to the official CMS repo.

### Proposed Changes

I attempted to simplify the pages/styling. Definitely worth going back on light sprints and seeing if it's possible to condense any of the code for simplicity.

*Note*: The links for _Log in_ and _Request Access_ are currently not set.

### Change Details

N/A

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### Feedback Requested

Please let me know if you see other ways we can simplify this code.

### CODE REVIEW

**Comment by dhgreene** [comment]
Thanks for adding those fonts!  Can you also investigate this console error I get loading the static site homepage but not from the web app?
Screen Shot 2020-04-08 at 12 33 08 PM

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,168 @@
+{
+  "name": "docker-jekyll-shell",
```
[the remark this answers is not in the document]
Comment (Sun-Mountain) [reply package-lock.json:2]: I'll pull your changes and run it locally to make sure.

**Comment by dhgreene** [comment]
> Thanks for adding those fonts! Can you also investigate this console error I get loading the static site homepage but not from the web app?
> Screen Shot 2020-04-08 at 12 33 08 PM

☝️ For future reference: this only appears when running Chrome locally from `localhost` (not `127.0.0.1`).  Not a problem!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+```
+docker-compose -f docker-compose.yml build static_site
+docker-compose -f docker-compose.yml run --rm static_site
+```
+
+This process uses a Docker container to execute `bundle exec jekyll build`, compiling site files into the same `_site` directory used when executing this command on the Docker host. The advantage here is that there's no need to install ruby or any dependencies on the machine building the static site—Docker takes care of all that.
+
+### Using Docker for serving
+To host the site in Docker, accessible at `http://localhost:4000/`:
```
Comment (dhgreene) [inline README.md:37]: To pair with the change a few lines below:
```suggestion
To host the site in Docker, accessible at `http://localhost:4001/`:
```

**Comment by Sun-Mountain** [comment]
> I notice that the content of the user guide on a grey background. Code snippets are also on a grey background, albeit a darker one. Don't know if that will present 508 compliancy errors, if not general accessibility issues, because I know that high contrast is a key tenet.
> 
> Other than that, language for the user guide looks good (though we _are_ taking a fine-toothed comb to this later, right? so not too fussed)

We'll be doing a more in depth look at the language before we launch. I'll look to see if I can change the background of the documentation.

**Pull request merged**